src/iris_api/api.py

```diff
@@ -2437,7 +2465,39 @@ def on_get(self, req, resp, app_name):
                                                                   AND `application_id` = %(application_id)s
                                                                   ORDER BY time_to_claim) as time_to_claim
                                                             WHERE (SELECT @row_id := @row_id + 1)
-                                                            BETWEEN @incident_count/2.0 AND @incident_count/2.0 + 1)'''
+                                                            BETWEEN @incident_count/2.0 AND @incident_count/2.0 + 1)''',
+            'pct_successful_sms_last_month': '''SELECT ROUND((SELECT count(*) FROM `message`
```

Comment:
can we rename these stats to associate them with twilio? To make this tracking feature truly optional and make it easy to add other vendor support in the future.